todo/src/Todo.d.ts

```diff
@@ -0,0 +1,22 @@
+import { Dispatch, SetStateAction } from "react";
+
+interface IFormData {
+  taskName: string;
+  description: string;
+  id: number;
+  date: Date;
+}
+
+interface TodoTaskProps {
+  task: IFormData;
+  description: string;
+  id: number;
+  date: Date;
+  deleteTask(taskDelete: number): void;
+  editTask( taskEditId: number,taskEditName:string,taskEditDescription:string): void;
```

Comment:
can you not just pass the task ? instead of passing its attributes ?

Reply:
Here I'm passing  all attributes because in edit function we are comparing newly edited data vs old data. While calling editTask I have passed id of the old task and newly edited task name and description of which I'm maintaining state.

Reply:
Updated the code. Now,I have passed only the task not it's attributes.